cypressPageGenerator/README.md

```diff
@@ -0,0 +1,14 @@
+# Cypress Page Generator
+## Premise
+Our cypress service config has become so big and so convoluted that making changes to it is very painful. It would be better if there was a way of automating the process of selecting simorgh urls from a pool of available urls. This script solves that problem. It takes a series of urls of different page types, it filters out non simorgh urls and then creates the cypress object schema. 
```

Comment:
I kind of think if this PR solves a problem then the description of the problem is no longer needed in the codebase and belongs more in the PR description.